example/.example.env

```diff
@@ -1,6 +0,0 @@
-SEQUELIZE_TYPE=postgres
```

Comment:
I removed example app because it takes too much to keep it updated. We are going to prioritise examples in `adminjs-example-app` and detailed documentation at `docs.adminjs.co`